src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={(todo.completed
```

Comment:
The className 'TodoInfo' is repeated in both conditions. Consider refactoring this to avoid repetition.